package.json

```diff
@@ -10,7 +10,7 @@
         "generate": "ts-node ./scripts/multisrc/generate.ts",
         "clearMulti": "ts-node ./scripts/clearMultisrc.ts",
         "less": "npx less ./test_web/static/css/index.less ./test_web/static/css/index.css",
-        "host": "host.sh"
+        "host": "chmod +x ./host.sh && ./host.sh"
```

Comment:
![image](https://github.com/LNReader/lnreader-sources/assets/86464880/1b94be80-a5d3-413c-b252-c893899595ec)
somehow my windows cant run this

Reply:
Try changing the operation concatenation symbol to `;` instead of `&&` and check if it'd work

Reply:
Can this be changed to ``bash ./host.sh``

Reply:
bash is not in windows

Reply:
Nom is weird especially when doing cross-platform stuff.
It uses cmd.exe on windows and bash on Linux, so IG we'll need to make a .bat file instead and run it in windows. Well still need to run chmod (for Linux) but make the script itself run regardless if chmod finished?

Reply:
![image](https://github.com/LNReader/lnreader-sources/assets/86464880/026b0f92-0b22-449a-bb9b-6f008d6fdd90)
doenst work event with `;`

Reply:
Ye, stupid cmd/sh distinction
I don't know why it works on my machine... Maybe I have changed the npm shell script and mine runs bash, but the default is cmd and that doesn't have chmod. So I guess we are stuck with two different scripts

Reply:
https://github.com/LNReader/lnreader-sources/pull/725/commits/e3e920f200fa8ea8f51ec851210836e1657199e7
Added both `host-linux` and `host-windows`